.gitignore

```diff
@@ -16,3 +16,4 @@ release/
 **/*.swp
 **/*.swo
 **/*-min.js
+**/package-lock.json
```

Comment:
Far be it from me to kick of a JS build systems discussion, but I think dependency-pinning is the right approach here, and shouldn't be abandoned.

Reply:
Noted. If I get more pushback on this, will revert that part of the change.

Reply:
I agree that these should probably be kept in place.  We want as much determinism and certainty as we can get in the Landscape builds, and the lockfiles help with that.

Reply:
Thanks for noting this, will revert that part of the change.

Reply:
Done.